CHANGELOG.md

```diff
@@ -8,10 +8,15 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## [2.9.1] - 2021-11-03
+
 ### Fixed
 
-- Hide enrollment count by default until an explicit minimum is set
 - Improve handling for autocompletion missing query errors
+- Stack organization logo to the right on course glimpse
+- Fix React course glimpse category badge to align with DjangoCMS glimpse
+- Hide enrollment count by default until an explicit minimum is set
+- Handle too-short query strings in <Search /> by displaying warning
```

Comment:
Whoops, there are mistakes in this changelog.

From my PR about course glimpse I'll add entry under `[2.9.0] - ...`. It seems there is also the same mistake for the search query fix.
So you should remove lines 31, 32 and 38.

```md
L31 - Fix rendering difference between React and Django course glimpses
L32 - Fix an error when a user typed a short search query in the header or home
...
L38 - Move organization logo behind category badge
```

Reply:
well spotted :clap:  I fixed it!